ia/src/process.py

```
@@ -14,15 +14,12 @@ def iaProcess(argv: list[str]) -> None:
-        # message: List[str] = ["coucou", "caca", "robin", "etienne"]
-        # crypted: List[Tuple[bytes, bytes]] = []
-        # decrypted: List[str] = []
-        # crypted = get_message(message)
-        # print("sentence crypted: ", crypted)
-        # decrypted = get_decrypted_message(crypted)
-        # print("sentence decrypted: ", decrypted)
-        # Parser = ia_parser(argv)
-        Parser = ia_parser(argv)
-        return Parser
-        
+        a = encrypt('I_need_help_to_level_up_to_X_with_Y')
```

Ruff (0.11.9)

`encrypt` may be undefined, or defined from star imports

(F405)